src/app/view/editor-main-view/data-views/trainrunsections.view.ts

```diff
@@ -1417,6 +1443,32 @@ export class TrainrunSectionsView {
           );
         }
       });
+
+    if (hasWarning) {
```

Comment:
This check is used for performance improvement: 
When the data has no warning - it will not add an empty svg / hmtl tag <svg:title></svg:title>. The filter (above) will remove all data with warning -> Rendering Pass 1 

When filtering (enable all data with warning) then the <svg:title> msg text <svg:title> will be append (inserted) into <svg:text>TEXT <svg:title> msg </svg:title></svg:text)

This was the most easy way to reduce the number of dom elements